src/main.rs

```diff
@@ -0,0 +1,76 @@
+use html5ever::serialize::{serialize, SerializeOpts};
+use std::borrow::Cow;
+use std::default::Default;
+use std::env;
+use std::ffi::OsStr;
+use std::io::{self, BufWriter};
+use std::path::{Path, PathBuf};
+
+use markup5ever_rcdom::SerializableHandle;
+
+mod annotate_attributes;
+mod boilerplate;
+mod dom_utils;
+mod interface_index;
+mod parser;
+mod represents;
+mod tag_omission;
+
+#[tokio::main]
+async fn main() -> io::Result<()> {
+    // Since we're using Rc in the DOM implementation, we must ensure that tasks
+    // which act on it are confined to this thread.
+
+    // Find the paths we need.
+    let cache_dir = path_from_env("HTML_CACHE", ".cache");
+    let source_dir = path_from_env("HTML_SOURCE", "../html");
```

Comment:
Maybe we should not have defaults here, and only specify the defaults in `build.sh`. WDYT?

Reply:
Selfishly, this makes quick uses of `cargo run` a little easier.